.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -771,17 +771,19 @@ async function installMetaCertificates(werft: Werft, branch: string, withVM: boo
     await installCertificate(werft, metaInstallCertParams, { ...metaEnv(), slice: slice });
 }
 
-async function installMonitoring(kubeconfig: string, namespace: string, nodeExporterPort: number, domain: string, stackdriverServiceAccount: any, withVM: boolean, observabilityBranch: string) {
-    const installMonitoringSatelliteParams = new InstallMonitoringSatelliteParams();
-    installMonitoringSatelliteParams.kubeconfigPath = kubeconfig
-    installMonitoringSatelliteParams.branch = observabilityBranch;
-    installMonitoringSatelliteParams.satelliteNamespace = namespace
-    installMonitoringSatelliteParams.clusterName = namespace
-    installMonitoringSatelliteParams.nodeExporterPort = nodeExporterPort
-    installMonitoringSatelliteParams.previewDomain = domain
-    installMonitoringSatelliteParams.stackdriverServiceAccount = stackdriverServiceAccount
-    installMonitoringSatelliteParams.withVM = withVM
-    installMonitoringSatellite(installMonitoringSatelliteParams);
+async function installMonitoring(werft: Werft, kubeconfig: string, namespace: string, nodeExporterPort: number, domain: string, stackdriverServiceAccount: any, withVM: boolean, observabilityBranch: string) {
```

Comment:
same thing here, we declare an `async` function only to use `await` when calling it. Would it make sense to just remove it?

Reply:
I'm keeping this async as it's my understand that we want to block in core-dev (when observability is enabled) but not block when installing it in VM-based preview environments ☺️